website/docs/r/container_group.html.markdown

```diff
@@ -134,6 +134,10 @@ A `container` block supports:
 
 * `memory` - (Required) The required memory of the containers in GB. Changing this forces a new resource to be created.
 
+~> **Note:** Gpu resources are currently only supported in Linux containers.
+
+* `gpu` - (Optional) Specify to deploy the container with a GPU resource.
```

Comment:
for consistency with the other resources could we make this:

```suggestion
* `gpu` - (Optional) A `gpu` block as defined below.
```

Reply:
Done, thanks.